PKGBUILD

```diff
@@ -7,7 +7,7 @@
 
 pkgname=gitkraken
 pkgrel=2
```

Comment:
> pkgrel=2

Why?

Reply:
Because I forget about it every single time.

Reply:
It would be good if you change it down to 1. :)

Reply:
Sure I will, just got my PC on